src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -550,6 +551,16 @@ def is_vault_secret(cmd, credential):
     return keyvault_dns.upper() in credential.upper()
 
 
+def get_task_id_from_task_name(cli_ctx, resource_group, registry_name, task_name):
```

Comment:
From the user's perspective for task run, will they ever know what this task ID is? Will GET for a TaskRunRequest show the Task ID, not the Task name?

Reply:
Yes, get shows both `id` and `name` like before.

```
**"GET /subscriptions/f9d7ebed-adbd-4cb4-b973-aaf82c136138/resourceGroups/docs/providers/Microsoft.ContainerRegistry/registries/taskdocs/tasks/btl-test?api-version=2019-06-01-preview HTTP/1.1" 200 None**

{"type":"Microsoft.ContainerRegistry/registries/tasks","properties":{"provisioningState":"Succeeded","creationDate":"2019-05-24T00:31:28.1775737+00:00","status":"Enabled","platform":{"os":"linux","architecture":"amd64"},"agentConfiguration":{"cpu":2},"timeout":3600,"step":{"type":"Docker","imageNames":["btl:{{.Run.ID}}"],"isPushEnabled":true,"noCache":false,"dockerFilePath":"Dockerfile-app","arguments":[{"name":"REGISTRY_NAME","value":"taskdocs.azurecr-test.io","isSecret":false}],"baseImageDependencies":[{"type":"Runtime","registry":"taskdocs.azurecr-test.io","repository":"baseimages/node","tag":"9-alpine","digest":"sha256:77fa392421a3a8bf86d05080e2dbf5016d7724d7e34552f52d5958ede452c458"}],"contextPath":"https://github.com/shahzzzam/acr-build-helloworld-node.git"},"trigger":{"sourceTriggers":[{"sourceRepository":{"sourceControlType":"Github","repositoryUrl":"https://github.com/shahzzzam/acr-build-helloworld-node.git","branch":"master"},"sourceTriggerEvents":["commit"],"status":"Enabled","name":"defaultSourceTriggerName"}],"baseImageTrigger":{"baseImageTriggerType":"Runtime","updateTriggerPayloadType":"Default","status":"Enabled","name":"defaultBaseimageTriggerName"}}},"id":"/subscriptions/f9d7ebed-adbd-4cb4-b973-aaf82c136138/resourceGroups/docs/providers/Microsoft.ContainerRegistry/registries/taskdocs/tasks/btl-test","name":"btl-test","location":"westus"}
```